android/modules/ui/src/java/ti/modules/titanium/ui/widget/tableview/TiTableViewRowProxyItem.java

```diff
@@ -506,21 +506,26 @@ protected void onMeasure(int widthMeasureSpec, int heightMeasureSpec)
 				}
 
 				if (height == null) {
-					h = Math.max(h, Math.max(content.getMeasuredHeight(), Math.max(leftImageHeight, rightImageHeight)));
-					h = Math.max(h, minRowHeight);
+				    // If measure spec is not specified, height should behave as Ti.UI.SIZE
+				    if (hMode == 0) {
```

Comment:
Probably better if in the future magic numbers not used if possible although comments have the info.
I.e. `0` when I assume it's `MeasureSpec.UNSPECIFIED`